### PR TITLE
Makes the megaphone let you scream bigly enough

### PR DESCRIPTION
**Pull request by ghost**
with '#' will be ignored, and an empty message aborts the commit.

[Changelogs]: Adds 3 cool new screams which are used when you scream with a megaphone

:cl: Megaphone screams!
add: 3 new ogg files to sounds/voices
del: Removed old things
tweak: edited megaphone.dm a bit
soundadd: added new screams for the megaphone
code: megaphone.dm
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Was also suggested in a forum post and accepted. please tell me if something is wrong.

### CODE REVIEW

File: code/game/objects/items/devices/megaphone.dm [hunk 1]
```diff
@@ -9,6 +9,7 @@
 	siemens_coefficient = 1
 	var/spamcheck = 0
 	var/list/voicespan = list(SPAN_COMMAND)
+	alternate_screams = list('hippiestation/sound/voice/scream_cowboy1.ogg', 'hippiestation/sound/voice/scream_cowboy2.ogg', 'hippiestation/sound/voice/scream_cowboy3.ogg')
```
Comment (ghost) [inline code/game/objects/items/devices/megaphone.dm]: modularise this please!
make a new file:
hippiestation/code/game/objects/items/devices/megaphone.dm

in that file do this
```
/obj/item/device/megaphone
    alternate_screams = list('hippiestation/sound/voice/scream_cowboy1.ogg', 'hippiestation/sound/voice/scream_cowboy2.ogg', 'hippiestation/sound/voice/scream_cowboy3.ogg')
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
also i was gonna link [this](https://www.youtube.com/watch?v=CCVdQ8xXBfk) before i actually even looked at the files)))

File: code/game/objects/items/devices/megaphone.dm (hunk 1 above)
Comment (JohnGinnane) [reply code/game/objects/items/devices/megaphone.dm]: i don't even think it'll work, i should check this out..

Comment (JohnGinnane) [reply code/game/objects/items/devices/megaphone.dm]: yeah i should total add the hand slots for this: https://github.com/HippieStation/HippieStation/blob/27e4596befb785dd2db7f75a7cd2a28f91cd8b68/hippiestation/code/modules/mob/living/carbon/reindex_screams.dm#L4

Comment (ghost) [reply code/game/objects/items/devices/megaphone.dm]: how do i exit the commit thing in git bash?

Comment (ghost) [reply code/game/objects/items/devices/megaphone.dm]: the virtual machine?
i usually fucking close the git bash terminal and reboot it lmfao

**Review by KayeArray — CHANGES_REQUESTED** [review]
Currently this won't work because the reindex_screams proc used by alternate_screams only works for clothes. You need to call add_screams() directly when the object is picked up and remove them when it is dropped

**Comment by ghost** [comment]
will do

**Comment by JohnGinnane** [comment]
@banthebantz I think will update the scream code this weekend to allow for this

**Comment by ghost** [comment]
oh cool

**Comment by Manly-Dwarf** [comment]
did you test this

**Comment by ghost** [comment]
waiting for reindex_screams refactor from john ginnane

**Comment by JohnGinnane** [comment]
i forgot to do the thing im sorry ((

**Comment by praisenarsie** [comment]
@JohnGinnane either close this or do the refactor!!!

**Comment by JohnGinnane** [comment]
i worked on it on weekend but stopped ((

**Comment by earthmoonjune** [comment]
+1

**Comment by Pyko1** [comment]
sure +1

**Comment by Jujumatic** [comment]
stale? it's been nearly 2 months now with no changes

**Pull request closed without merging**